offload_vio/device_tx/CMakeLists.txt

```diff
@@ -10,7 +10,7 @@ find_package(eCAL REQUIRED)
 find_package(Protobuf REQUIRED)
 find_package(OpenCV REQUIRED)
```

Comment:
What would happen if the user doesn't have the exact version?

Reply:
sometime if your library is compiled using different OpenCV it will run into error

Reply:
Is this the version CMake #384 will install? Otherwise we cannot assume that the user has this OpenCV.

Reply:
I'll change to to `find_package(OpenCV 4 REQUIRED)` since that's the one to be used in the cmake version anyway.

Reply:
You are right. I was still thinking about our earlier default master, which uses 3.4.6. Given we are supporting five versions of OS, please make sure that all default OpenCV version is over 4.0.

Reply:
Giving it another thought, I think we can require 3.4.6 because that's what's installed by our current dependencies installing script. The cmake version can then ask for 4 if it's installing 4 as the dependency. Comments?

Reply:
For any supported system (e.g. Ubuntu 20.04 and 22.04) you get OpenCV 4 by default. Also 3.4.6 and 4.0+ have different include files and some API differences so you would need to add include guards. For every other plugin we are only supporting 4.0+ Also I believe 3.4.6 is deprecated.

Reply:
Thanks for the inputs. Hopefully the cmake version will be ready soon and this will be out of concern. But before that, it's just weird to me if we install 3.4.6 but require 4. Changing the installing script to install OpenCV4 will, as you said, cause #include issues that don't worth the time fixing. I guess I will stick to 4 here. If the users do hit some errors it should be a easy fix. Let me know if you have other thoughts.